### ISSUE TITLE
add Team Player Colors & U.I form, in OpenRA (for observer and not only)

### ISSUE THREAD

**Issue by JohnnyXP64** [OpenRA/OpenRA#7497]
Dear all (specially devs).
After watching some of the SoScared101 match reply casts, I was surprised that I saw a very important and useful feature is missing from the OpenRA engine. “Friend or Foe”(FoF) coloring support and a small TEAM User Interface form.
Right now it’s a chaos to understand who is an ally with whom, who is the enemy, who allied maybe during a match with another etc.
5 simple an elegant ideas to solve that, and promote more the OpenRA engine to the multiplayer community allowing easier podcasting of a match, are:

1)have a button for the Observer or the person you views the reply, to change the players selected colors in units, structures, and minimap colors that are the same if they are on the same team! Let’s say we have a 2 vs 2 vs 2 match. The 2 players could be blue, the other 2 yellow and the rest green.

2)have the player and the observer to change the color of the selected player to “FoF”, Green from himself, Yellow for all his allies, and Red for all his Enemies.

3)in addition to the first 2 ideas, just make sure the player’s name, country-side flag it’s very clear when the mouse is going over their units, as a tooltip. (as it is right now)

4)Make a small form at the free bottom left corner of the screen accessible from the observer or the player during the game, where you can see always on top of your screen the current player names, side flags, colors, but most important group them by teams, for a fast and easy understanding of the current battle contentions.

5)Make an option “Team color Selection” into the lobby when creating a game. Allow players to select to be in TEAM 1 (BLUE) TEAM 2 (RED) let them pick a color for their team and their units should be shown as this color when they choose to switch between “FoF” coloring mode in game.
Any of those, standalone or as a combination or something like that could be used to help us enjoy more the OpenRA games.
Thanks for your time, and I hope you see how helpful and needed is Team Player Colors in OpenRA.

Regards
JohnnyXP64

**Comment by SoScared** [comment]
:+1: Team games with lots of players could benefit such a feature, especially 1) and 2). The majority of online games seem to be played on big maps with big teams.

In Starcraft 2 you can toggle on FoF colors for the minimap which works nicely. A feature for the minimap alone could be very helpful!

For each openRA game there's almost always new set of colors in play, sometimes obscure, that makes a big team game a bit hard to follow. I'm sure a lot of players would be grateful to get some type of team-color option in the lobby, or better yet, in-game.

**Comment by Mailaender** [comment]
In OpenRA you can currently enable "Team Health Colors" and "Always Show Status Bar" which seems to be the DOTA style. https://github.com/OpenRA/OpenRA/pull/4054

Looks like I closed https://github.com/OpenRA/OpenRA/issues/2462 too early.

**Comment by Mailaender** [comment]
See https://github.com/OpenRA/OpenRA/issues/7538 for a proposed fix.

**Comment by Mailaender** [comment]
Duplicate of https://github.com/OpenRA/OpenRA/issues/2462
